.builders/deps/host_dependencies.txt

```diff
@@ -1 +1,2 @@
+google-cloud-storage
```

Comment:
@ofek should we be worried that this isn't pinned? I'm not the most experienced dev, but so far every time I've seen an unpinned build or CI dependency, it's blown up at one point or another.

Reply:
Yes we should! I'll do that when I confirm everything is working.

Reply:
coolio! just figured I'd ask before I forget and while this is fresh in my notification stream 🙈